appservice/src/createAppService/SiteHostingPlanStep.ts

```diff
@@ -6,17 +6,24 @@
 import { AzureWizardPromptStep, IAzureQuickPickItem, IWizardOptions } from 'vscode-azureextensionui';
 import { ext } from '../extensionVariables';
 import { localize } from '../localize';
+import { WebsiteOS } from './AppKind';
 import { AppServicePlanListStep } from './AppServicePlanListStep';
 import { IAppServiceWizardContext } from './IAppServiceWizardContext';
 
 export class SiteHostingPlanStep extends AzureWizardPromptStep<IAppServiceWizardContext> {
     public async prompt(wizardContext: IAppServiceWizardContext): Promise<void> {
         const placeHolder: string = localize('selectHostingPlan', 'Select a hosting plan.');
-        const picks: IAzureQuickPickItem<boolean>[] = [
-            { label: localize('consumption', 'Consumption Plan'), data: true },
-            { label: localize('dedicated', 'App Service Plan'), data: false }
+        const picks: IAzureQuickPickItem<[boolean, RegExp | undefined]>[] = [
```

Comment:
Would it make sense to make an enum that has the different RegExps that would be used as a filter? 

(E is elastic and EP is elastic premium?)

Reply:
Not sure what you mean. Are you saying an enum made up of RegExps? I don't think that's a thing

Also I don't think this is a common scenario, so no I don't think it's worth it